packages/components/src/components/as-histogram-widget/as-histogram-widget.tsx

```diff
@@ -16,8 +16,10 @@ import { Container } from './types/Container';
 import dataService from './utils/data.service';
 import drawService from './utils/draw.service';
 
-const DEFAULT_BAR_COLOR = 'var(--as--color--primary, #1785FB)';
-const DEFAULT_SELECTED_BAR_COLOR = 'var(--as--color--complementary, #47DB99)';
+const DEFAULT_BAR_COLOR_HEX = '#1785FB';
```

Comment:
Is it impossible to get this color from the `scss` file? It'd be great to have all colors sync'ed.

At least add a comment to tell that it's the primary color.

Reply:
👍